src/file.ts

```diff
@@ -93,3 +107,47 @@ export async function getEntryData(
   }
   return { data: entry.data };
 }
+
+// ===============
+// Encrypted Files
+// ===============
+
+/**
+ * Gets Encrypted JSON set with MySky at the given data path for the given
+ * public user ID.
+ *
+ * @param this - SkynetClient
+ * @param userID - The MySky public user ID.
+ * @param pathSeed - The share-able secret path seed.
+ * @param [customOptions] - Additional settings that can optionally be set.
+ * @returns - An object containing the decrypted json data.
+ */
+export async function getEncryptedJSON(
```

Comment:
the method is a little misleading no, `getEncryptedJSON` returns an object containing decrypted json data

Reply:
Going to try to start a discussion about it.

Reply:
> I understand the objection, but I'm not sure of a better name aside from like getDecryptedEncryptedJSON, which seems annoying and overly descriptive.
- @dghelm

Reply:
Changed the name to follow the rest of our API where we suffix qualifiers, i.e. `getJSONEncrypted` to match `downloadHns`.